int-8018x.c

```diff
@@ -11,32 +11,89 @@
 
 #define INT_REG_EOI  0
 #define INT_REG_MASK 3
+#define INT_REG_PRIMSK 4
+#define INT_REG_REQST 6
+#define INT_REG_TCUCON 8
+#define INT_REG_SCUCON 9
+
+#define INT_REG_I4CON 10
+#define INT_REG_I0CON 11
+#define INT_REG_I1CON 12
+#define INT_REG_I2CON 13
+#define INT_REG_I3CON 14
 
 int _int_line_max = INT_LINE_MAX;
 int _int_prio_max = INT_PRIO_MAX;
 
 // Timer & serial are edge triggered
 
+int _pri_mask = 7;
+
 int _int_mode [INT_LINE_MAX] =
-	{ 1, 0, 1, 0, 0, 0, 0, 0};
+	{ 1, 0, 1, 0, 0, 0, 0, 0, 0, 1, 1};
 
 int _int_prio [INT_LINE_MAX] =
-	{ 0, 1, 2, 3, 4, 5, 6, 7};
+	{ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
 
 int _int_vect [INT_LINE_MAX] =
-	{ 0x08, 0x09, 0x0A, 0x0B, 0x0C, 0x0D, 0x0E, 0x0F };
+	{ 8, 0, 20, 17, 12, 13, 14, 15, 21, 18, 19 };
```

Comment:
Are you sure of that vector table ? According to the Intel 80188 datasheet:
```
 8 -> Timer0
 9 -> Reserved
10 -> DMA0
11 -> DMA1
12 -> INT0
13 -> INT1
14 -> INT2
15 -> INT3
16 -> ?
17 -> ?
18 -> Timer1
19 -> Timer2
```

Reply:
I was doing the changes according to 80186EB datasheet.

The defines of individual "lines" have been extended by TIMER1,TIMER2, SERIAL RX
![shot-2023-12-02_01-19-17](https://github.com/mfld-fr/emu86/assets/204234/90ec22c1-bd03-4a1f-b343-c3c648b294a9)

And the line to vector mapping are from 80186EB datasheet. 
![shot-2023-12-01_23-15-35](https://github.com/mfld-fr/emu86/assets/204234/b58a78f1-97c5-498a-8b6b-855af1dad4b3)

Which is consistent with your table from 80188 table. (except 80186 has no DMA and 80188 has no serial port(?))

Reply:
Thanks for that 'EB' vector table that clarifies. It looks like the initial numbering of the interrupt lines was not correct before your PR, leading to confusion. For example, Serial RX is not wired on line 2 (vector 10) , but on 12 (vector 20). I will fix that numbering after merging your PR.

@cocus already raised the point, but I understand from this conversation that this project should definitively document the differences between the 8018x variants. For example, it appears that 'EB' has no DMA, while 'XL', 'EA' and 'EC' have one, and it has a serial port, while 'XL' and 'EA' have not.

Reply:
With the other 8018x in mind, it would make sense to refactor the interrupt vector table and line defines to have consistent ordering.

Reply:
Thanks for the datasheet.